Rover/commands.cpp

```diff
@@ -33,7 +33,7 @@ bool Rover::set_home(const Location& loc, bool lock)
     mode_auto.mission.write_home_to_storage();
 
     // send text of home position to ground stations
-    gcs().send_text(MAV_SEVERITY_INFO, "Set HOME to %.6f %.6f at %.2fm",
+    gcs().send_text(MAV_SEVERITY_INFO, "Set HOME to %.7f %.7f at %.2fm",
```

Comment:
this won't work on ChibiOS at least, possibly not on SITL too, we go via a float inside the printf engine